views/build/grunt/bundle.js

```diff
@@ -1,53 +1,42 @@
-module.exports = function(grunt) {
-
-    var requirejs   = grunt.config('requirejs') || {};
-    var clean       = grunt.config('clean') || {};
-    var copy        = grunt.config('copy') || {};
-
-    var root        = grunt.option('root');
-    var libs        = grunt.option('mainlibs');
-    var ext         = require(root + '/tao/views/build/tasks/helpers/extensions')(grunt, root);
-    var out         = 'output';
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015-2018 (original work) Open Assessment Technologies SA;
+ */
 
-    /**
-     * Remove bundled and bundling files
-     */
-    clean.taobackofficebundle = [out];
+/**
+ * Configure the extension bundles
+ * @author Bertrand Chevrier <bertrand@taotesting.com>
+ */
+module.exports = function(grunt) {
+    'use strict';
 
-    /**
-     * Compile tao files into a bundle
-     */
-    requirejs.taobackofficebundle = {
-        options: {
-            baseUrl : '../js',
-            dir : out,
-            mainConfigFile : './config/requirejs.build.js',
-            paths : {
-                'taoBackOffice' : root + '/taoBackOffice/views/js',
-                'taoBackOfficeCss' : root + '/taoBackOffice/views/css'
-            },
-            modules : [{
-                name: 'taoBackOffice/controller/routes',
-                include : ext.getExtensionsControllers(['taoBackOffice']),
-                exclude : ['mathJax', 'taoBackOffice/lib/vis/vis'].concat(libs)
-            }]
+    grunt.config.merge({
+        bundle : {
+            taobackoffice : {
+                options : {
+                    extension : 'taoBackOffice',
+                    bundles : [{
```

Comment:
no need for `outputDir : 'loader'` here ?

Reply:
it's the default value too, but will add it for consistency

Reply:
ok